src/index.html

```diff
@@ -7,6 +7,361 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logoLink">
+        <img
+          src="/images/logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
```

Comment:
more than 3 attributes have to be in separate line

Reply:
There is only 2 attributes...

Reply:
href - 1, class - 2, is-activ - 3

Reply:
omg, sorryyyyyy!